scripts/process-queue.ts

```
@@ -0,0 +1,75 @@
+    const toProcess = rps === -1 ? rpsQueue.length : Math.min(rps, rpsQueue.length);
+    console.log(`[Queue] Processing ${toProcess} per-second queued requests for model ${modelId}`);
+    for (let i = 0; i < toProcess; i++) {
+      const item = await redis.lpop(rpsQueueKey);
+      if (!item) break;
+      const req = JSON.parse(item);
+      await handleRequest(req, model);
+    }
+  }
+}
+
+async function handleRequest(req: any, model: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)